integration-tests/java/README.md

```diff
@@ -28,4 +28,42 @@ You can run tests from the top level directory with the following command.
 
 ```bash
 $ ./gradlew integration-tests:java:integrationTest
-``` 
+```
+
+# How to initiate a new project from this example
+This section explains how you can write your Java clients based on this example.
+
+### Set up a new directory
+Create a new directory, and copy entire code Java integration tests to there.
```

Comment:
Create a new directory, and copy the entire contents of the java integration directory into it.